qfieldsync/gui/layers_config_widget.py

```diff
@@ -143,6 +174,13 @@ def reloadProject(self):
                 cmb, available_actions, self.get_layer_action(layer_source)
             )
 
+            # Save the current action for the layer
+            cmb.currentIndexChanged.connect(
+                lambda index, l_source=layer_source: self.set_layer_action(
+                    l_source, cmb.itemData(index)
+                )
+            )
+
```

Comment:
Is this related to this PR?  If not, can we make a separate PR with it. Also, please make it a separate function, the lambda is a bit hard to read.

Reply:
Yes, it is related. It is added to save the action on the layers.